consensus/core/src/metrics.rs

```diff
@@ -19,15 +21,24 @@ const LATENCY_SEC_BUCKETS: &[f64] = &[
 pub(crate) struct Metrics {
     pub(crate) node_metrics: NodeMetrics,
     pub(crate) channel_metrics: ChannelMetrics,
+    pub(crate) inbound_network_metrics: NetworkMetrics,
```

Comment:
It seems inbound_network_metrics and outbound_network_metrics agnostic to the underlying network implementation? Maybe wrap them in a `network_metrics` field, so we can have places for implementation and direction agnostic metrics in future, e.g. connected peers, rtt latencies etc.